app/views/reviews/new.html.erb

```diff
@@ -0,0 +1,22 @@
+<h1>Add a Review</h1>
+
+  <%= form_for [@item, @review] do |f| %>
+    <%= f.label :rating %>
+    <%= f.radio_button :rating, 1 %>
+    <%= f.label :rating_1, 1 %>
+    <%= f.radio_button :rating, 2 %>
+    <%= f.label :rating_2, 2 %>
+    <%= f.radio_button :rating, 3 %>
+    <%= f.label :rating_3, 3 %>
+    <%= f.radio_button :rating, 4 %>
+    <%= f.label :rating_4, 4 %>
+    <%= f.radio_button :rating, 5 %>
+    <%= f.label :rating_5, 5 %>
```

Comment:
The `collection_radio_buttons` helper method could help you out, here: http://stackoverflow.com/a/18526150/2675670